dependabot-automerge/action.yml

```diff
@@ -25,12 +25,17 @@ inputs:
     default: ${{ github.token }}
   update-type:
     description: |
-      The highest level of update that can be automerged. The default value is
-      `patch`, and supported values are `major`, `minor`, and `patch`. It is not
-      recommended to specify `major`.
+      The highest level of update that can be automatically merged. The default
+      value is `patch`; supported values are `major`, `minor`, and `patch`.
```

Comment:
(nit, no action required, came to mind because I’m having lunch with one of my tech writing teachers tomorrow) From a technical writing perspective semicolons fall into the “avoid” category if you think your audience is not up to your standard of punctuation. That category also includes latin abbreviations because many readers either don’t know what they mean or can confuse their meanings. For example, many readers get the meanings of i.e. and e.g. muddled up.